### ISSUE TITLE
Should improve export support for svg and csv 

### ISSUE THREAD

**Issue by ip1200** [sgsinclair/Voyant#238]
The export function seems to be shrunk seriously compared with v1.0. Take Links for example, now I can only save a png on my disk, but in 1.0 I can get a svg which is editable in Adobe Illustrator--the png is too small to see clearly what's inside. Other tools are the same. Like Trends, 1.0 can export a plain text or a csv text or a svg, but now can only export a png or a html
I really hope you can bring the powerful export function in 1.0 version back completely, in fact I hope the export function can be more powerful (export the document-term matrix with some parameters given by the users and the distance matrix as csv file) as some of the information is really useful and can be further explored with other tools like SPSS.

**Comment by sgsinclair** [comment]
Thanks for the comment. Links in V2 uses a canvas-based library instead of an SVG-based library, but it should be possible to convert with something like http://gliffy.github.io/canvas2svg/ The new Links is awkward in smaller windows, but the layout algorithm and styling options are better.

Please feel free to spawn new issues for export functionality from specific tools.

**Comment by ip1200** [comment]
I don't know how to put it into action, more clear instruction?

**Comment by sgsinclair** [comment]
Right, for export of document term matrix, which tool were you thinking?

**Comment by ip1200** [comment]
IBM SPSS, I think, which is easier for beginners.

**Comment by ip1200** [comment]
Besides, I have no idea how the canvas2svg can work.
